test/common/bond_test.js

```diff
@@ -44,6 +48,6 @@ test('determineBondProvider should return the default BondProvider if the URL ho
     t.is(determineBondProvider('drs://some-host/anything'), BondProviders.default);
 });
 
-test('determineBondProvider should return the "jade-data-repo" as the provider for JDR host"', (t) => {
-    t.is(determineBondProvider('drs://jade.datarepo-dev.broadinstitute.org/identifier'), BondProviders.JADE_DATA_REPO);
+test('determineBondProvider should not return a provider if the URL host is JDR"', (t) => {
+    t.falsy(determineBondProvider('drs://jade.datarepo-dev.broadinstitute.org/identifier'));
 });
```

Comment:
Can we add more tests with Anvil and Kids First DRS uri here to check they return expected Bond providers?